File: app/src/main/java/com/hanstcs/movieapp/movies/MoviesActivity.java
package com.hanstcs.movieapp.movies;

import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.Toolbar;

import com.hanstcs.movieapp.R;

public class MoviesActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_movies);

        Toolbar toolbar = findViewById(R.id.toolbar);
        toolbar.setTitle(R.string.app_name);
        setSupportActionBar(toolbar);

        MoviesFragment moviesFragment =
                (MoviesFragment) getSupportFragmentManager().findFragmentById(R.id.contentFrame);
        if (moviesFragment == null) {
            moviesFragment = MoviesFragment.newInstance();
            addFragmentToActivity(moviesFragment);
        }
    }

    private void addFragmentToActivity(MoviesFragment moviesFragment) {
        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
        transaction.add(R.id.contentFrame, moviesFragment);
        transaction.commit();
    }
}

File: app/src/main/java/com/hanstcs/movieapp/movies/MoviesActivity.kt
package com.hanstcs.movieapp.movies

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.Toolbar

import com.hanstcs.movieapp.R

class MoviesActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_movies)

        val toolbar = findViewById<Toolbar>(R.id.toolbar)
        toolbar.setTitle(R.string.app_name)
        setSupportActionBar(toolbar)

        var moviesFragment =
                supportFragmentManager.findFragmentById(R.id.contentFrame)
        if (moviesFragment == null) {
            moviesFragment = MoviesFragment()
            addFragmentToActivity(moviesFragment);
        }
    }

    private fun addFragmentToActivity(moviesFragment: MoviesFragment) {
        val transaction = supportFragmentManager.beginTransaction()
        transaction.add(R.id.contentFrame, moviesFragment)
        transaction.commit()
    }
}
